integration_test/testgenerated/server/server-service.yml

```diff
@@ -1,4 +1,8 @@
 types:
+  imports:
+    Safe:
+      external:
+        java: com.palantir.logsafe.Safe
```

Comment:
There is already a unit test for the invalid import, but it might be nice to add another import with a different path and use that as a marker below to ensure it doesn't get processed as safe.